experiment_api/experiment.py

```diff
@@ -62,51 +67,33 @@
 import re
 import sys
 
-
-class Veri(object):
-    """ Structure of the Veri dataset unzipped and miscellaneous information
-    """
-    name_query_filepath = "name_query.txt"
-    name_test_filepath = "name_test.txt"
-    name_train_filepath = "name_train.txt"
-    image_query_filepath = "image_query"
-    image_test_filepath = "image_test"
-    image_train_filepath = "image_train"
-    ground_truth_filepath = "gt_image.txt"
-    junk_images_filepath = "jk_image.txt"
-    train_label_filepath = "train_label.xml"
-    num_cars = 776
-    num_cams = 20
-    num_query_images = 1679
-    num_test_images = 11580
-    num_train_images = 37779
-    total_images = 49358
+from utils import *
 
 
 class ExperimentGenerator(object):
-    def __init__(self, veri_unzipped_path, num_cams, num_cars_per_cam, drop_percentage, seed, time, otherMetric):
-        # set inputs
+    def __init__(self, veri_unzipped_path, num_cams, num_cars_per_cam, drop_percentage, seed, time, typ):
+        # set inputs         
```

Comment:
Is your pre-commit hook set up correctly? I think it should strip trailing spaces.